src/backend/tests/change-requests.test.ts

```diff
@@ -24,39 +25,81 @@ describe('Change Requests', () => {
   describe('reviewChangeRequest', () => {
     test('reviewer not found errors', async () => {
       jest.spyOn(prisma.user, 'findUnique').mockResolvedValue(null);
-      const response = await request(app).post('/review').send(whipPayloadObject);
-
-      expect(response.body).toStrictEqual({ message: `User with id #${whipPayloadObject.reviewerId} not found` });
-      expect(response.status).toBe(404);
+      try {
+        await ChangeRequestService.reviewChangeRequest(
+          whipPayloadObject.reviewerId,
+          whipPayloadObject.crId,
+          whipPayloadObject.reviewNotes,
+          whipPayloadObject.accepted,
+          '1'
+        );
+      } catch (e) {
+        // eslint-disable-next-line jest/no-conditional-expect
+        expect(e).toBeInstanceOf(NotFoundException);
+        // eslint-disable-next-line jest/no-conditional-expect
+        expect(e).toEqual(new NotFoundException('User', 1));
+      }
       expect(prisma.user.findUnique).toHaveBeenCalledTimes(1);
     });
 
     test('reviewer doesnt have access errors', async () => {
       jest.spyOn(prisma.user, 'findUnique').mockResolvedValueOnce(wonderwoman);
-      const response = await request(app)
-        .post('/review')
-        .send({ ...whipPayloadObject, accepted: false });
-      expect(response.body).toStrictEqual({ message: 'Access Denied' });
-      expect(response.status).toBe(403);
+
+      try {
+        await ChangeRequestService.reviewChangeRequest(
+          whipPayloadObject.reviewerId,
```

Comment:
now that we dont have the request bodys, we don't need payload objects at all and can just use variables

Reply:
with stuff like this, try to see what you can abstract and declare outside of the tests so you're not rewriting tons of the same variables in the code over and over